.travis.yml

```diff
@@ -19,4 +19,4 @@ script:
   - make nc-release
   - make tests
   - ./run-tests db-only shredding
-  - ./run-tests lens
+  - ./run-tests unit 
```

Comment:
oUnit has a hierarchical structure, adding more tests can be done by adding another test suite to the root of the tests https://github.com/links-lang/links/blob/master/tests/lens/unitTests.ml. Requiring more effort and projects and messing around with bash scripts to run tests will just discourage people from using them.

Reply:
I disagree with you. My counter-arguments are per extension modularity, locality, and separation of concerns.

Maybe having a single master file like `unitTest.ml` is OK. But at the very atleast I think you should either put the lens-related modules into a separate directory or merge them into one big self-contained test module (some easily parsed name like `lensTests.ml`).

Reply:
I think it would be good for an extension such as lenses (or sessions, or effects/handlers, all of which change the language in hard-to-modularize ways) to be modular, and for the associated tests to be modular, but as far as I know we don't know how to do the former, so the latter seems like a lower-order concern.  Can we accept this approach for now and make an issue to try to find a better way?